Client/Frontend/Browser/TabManager.swift

```diff
@@ -518,84 +518,86 @@ class TabManager: NSObject {
     }
 }
 
-extension TabManager {
+class SavedTab: NSObject, NSCoding {
```

Comment:
Be careful with this change. I've had an issue in the past where after moving `SavedTab` from inside a class/extension scope to the file scope the app was no longer able to decode the tab from disk because technically it becomes a different class (`TabManager.SavedTab` != `SavedTab`) from the decoders point of view. The result of this is when users first launch the app from the change their session is gone because the decoding failed but then works on subsequent launches. We'll want to try to keep this class the same.